src/Webapi/Dom/Webapi__Dom__HtmlFormControlsCollection.resi

```diff
@@ -0,0 +1,17 @@
+type t = Dom.htmlFormControlsCollection
+
+@val @scope(("Array", "prototype", "slice")) external toArray: t => array<Dom.element> = "call"
+@get external length: t => int = "length"
+@send @return(nullable) external item: (t, int) => option<Dom.element> = "item"
+
+type t_namedItem =
+  | RadioNodeList(Dom.radioNodeList)
+  | Button(Dom.htmlButtonElement)
+  | FieldSet(Dom.htmlFieldSetElement)
+  | Input(Dom.htmlInputElement)
+  | Object(Dom.htmlObjectElement)
+  | Output(Dom.htmlOutputElement)
+  | Select(Dom.htmlSelectElement)
+  | TextArea(Dom.htmlTextAreaElement)
+
+let namedItem: (t, string) => option<t_namedItem>
```

Comment:
I'm not used to seeing interface files in this project but I guess it makes sense for what this file is doing 😅